pkg/pods/pod_test.go

```diff
@@ -224,7 +224,8 @@ func TestBuildRunitServices(t *testing.T) {
 		path:           "/data/pods/testPod",
 		ServiceBuilder: serviceBuilder,
 	}
-	hl := hoist.FakeHoistLaunchableForDir("multiple_script_test_hoist_launchable")
+	hl, sb := hoist.FakeHoistLaunchableForDir("multiple_script_test_hoist_launchable")
+	defer os.RemoveAll(sb.RunitRoot)
```

Comment:
I sympathize with the desire to keep a small public interface, but I see some latent danger here. If the implementation of `FakeHoistLaunchableForDir()` ever changes, there's a stray `os.RemoveAll()` that might remove more than anticipated. Having one function that is dedicated to cleaning up after `Fake...()` keeps all the cleanup code in one place.